docs/DeveloperGuide.md

```diff
@@ -1110,4 +1110,95 @@ testers are expected to do more *exploratory* testing.
 
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
-1. _{ more test cases …​ }_
+2. _{ more test cases …​ }_
+
+### Adding a note
+
+1. Adding a note with title that is yet to exist
+   1. Test case: `addNote title/event content/november 3rd 4pm`
+      Expected: Note with title `event` and content `november 3rd 4pm` added into notes list. Status bar displays message stating new note is added. Notes display panel shows new note.
```

Comment:
This is the called the Result Display, the status bar is a footer element that has been removed